### ISSUE TITLE
Training datasets for different applications

### ISSUE THREAD

**Issue by XinranQin** [Vandermode/TFPnP#3]
As the paper said “For the CS-MRI application, a single policy network is trained to handle multiple sampling ratios (with x2/x4/x8  acceleration) and noise levels (5/10/15), simultaneously.  Similarly, one policy network is learned for phase retrieval under different settings. “, does the motioned training datasets of applications (MRI, PR, CT, SPI) are all PASCAL VOC dataset?  Because there is a domain shift between the training datasets and the testing dataset in (MRI, CT, SPI).

**Comment by Vandermode** [comment]
Nope, we use the same dataset (PASCAL VOC) to learn policy for all applications. Since the policy network is only responsible to the parameter selection within the PnP algorithm (the learning space is highly regularized), we don't find severe domain shift issue related to image modalities, in contrast to pure DL-based image reconstruction approaches. But you are welcome to use other training data matching the test image modalities:)

**Comment by XinranQin** [comment]
Thanks for your reply
